docs/user-guide/coordinate-transformations.ipynb

```diff
@@ -287,8 +287,9 @@
     "                      'location': location,\n",
     "                      'datetime': datetime\n",
     "                  })\n",
-    "da += 2. * (location.fields.z > 0. * sc.Unit('km')).astype(\n",
-    "    'float64')  # more lightning strikes in northern hemisphere\n",
+    "north = location.fields.z > 0. * sc.Unit('km')\n",
+    "north.unit = ''\n",
```

Comment:
What does that do? Does it set the unit to 'one'? If so, better spell it out, I think.